src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -35,28 +36,31 @@ $-control-size: 32px;
 
 .VideoHuddleControls {
   @include sparkle.panel--normal;
+  width: 5%;
```

Comment:
why 5% ? in figma we have width in px, where did you get 5 and 10 %?
can't you specify just min-width / max-width for it to avoid the situation like on the screenshot?
<img width="80" alt="image" src="https://user-images.githubusercontent.com/63313732/169270473-13b72264-6be7-4649-8008-317ca20946da.png">